server/src/main/java/com/decathlon/ara/Messages.java

```diff
@@ -55,7 +55,6 @@
     public static final String NOT_UNIQUE_PROBLEM_DEFECT_ID = "The defect ID is already assigned to another problem.";
     public static final String NOT_UNIQUE_PROBLEM_NAME = "The name is already used by another problem.";
     public static final String NOT_UNIQUE_PROJECT_CODE = "The code is already used by another project.";
-    public static final String NOT_UNIQUE_PROJECT_DEFAULT_AT_STARTUP = "There is already another default project.";
     public static final String NOT_UNIQUE_PROJECT_NAME = "The name is already used by another project.";
```

Comment:
I'm ok with the message removal, therefore it can be good to display a popup in the frontend with a confirmation request to the user before actually change the value (you know, miss clic and this kind of stuff...)

See the issue #162 for an example of message we can ask :)